### DISCUSSION TITLE
List sort order on the Your Stars page seems arbitrary

### DISCUSSION THREAD

**Discussion by tedmiston** [community/community#9592] [Other Feature Feedback, Questions, & Ideas]
<img width="925" alt="image" src="https://user-images.githubusercontent.com/366688/147966322-2ea89f53-998a-46ae-8cc6-275d272f87fb.png">

What exactly is the sort order of list names on the *Your Stars* page?

The lists don't seem to be sorted by creation date or alphabetically.  [I created the 2 Gatsby lists first, and then the 5 "tech to try" lists in the order of: highest, high, medium, low, lowest; hoping that that would lead to them being sorted that way.]

(See my screenshot above of https://github.com/tedmiston?tab=stars.)

I'm attempting to "namespace" my groups of lists with a leading emoji so that related lists are grouped together.

For this purpose, being able to rearrange the order of the "list" of lists would be a good enough solution as well.

Related:
- https://github.com/github/feedback/discussions/9199
- https://github.com/github/feedback/discussions/8921

**Comment by brianlovin** [comment]
Thanks for the note — right now they are ordered by _most recently active_ (e.g. when a repository was added to a list). I can see how this could be confusing or frustrating if you are trying to group things specifically!
